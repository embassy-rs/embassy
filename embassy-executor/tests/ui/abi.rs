#![cfg_attr(feature = "nightly", feature(impl_trait_in_assoc_type))]

struct Foo<'a>(&'a ());

#[embassy_executor::task]
async extern "C" fn task() {}

fn main() {}
