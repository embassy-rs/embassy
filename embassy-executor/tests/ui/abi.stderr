error: task functions must not have an ABI qualifier
 --> tests/ui/abi.rs:6:1
  |
6 | async extern "C" fn task() {}
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^
